packages/@okta/vuepress-site/docs/concepts/manage-user-creds/index.md

```diff
@@ -0,0 +1,190 @@
+* Insecure token transmission: All interactions and redirects involving users, apps, and Okta must be secured through HTTPS (SSL/TLS) connections to protect credentials and tokens from interception. Pass access tokens exclusively through an HTTP Authorization header. Also, never encode tokens into a payload or URL, as these could be logged or cached.
+<!-- [For later: The Okta Client SDK's token management system is designed to mitigate these issues and ensure app stability.] -->
+
+### Token lifecycle
+
+* Lifetime: Tokens from the Okta org authorization server have fixed lifetimes:
+  * ID token: 60 minutes
+  * Access token: 60 minutes
+  * Refresh token: 90 days
+  A custom authorization server allows configurable lifetimes for access tokens (5 minutes to 24 hours). It also allows configurable lifetimes for refresh tokens (minimum 10 minutes idle up to a maximum of 5 years). However, ID tokens remain set at 60 minutes.
+* Expiration and renewal: Refreshing expired access tokens is a balancing act between security and scalability. Shorter access token lifetimes are a strong security practice that limit the window of opportunity for attackers to use a compromised token. This increased security comes at the cost of requiring more frequent token refresh requests, which can create a scalability challenge for the identity provider (IdP). A refresh token is optional but recommended as they make it more functional to have access tokens with shorter lifetimes.
+* Revocation: When a token is no longer needed (for example, during sign-out), it's crucial to revoke it on the server using the Okta revocation endpoint. This is in addition to clearing it from the client. Revocation of a token can also happen implicitly due to user deactivation, client app changes, or configuration changes of the authorization server.
+* Validation: You can validate access tokens either locally (by verifying the JWT signature using JWKS public keys) or through introspection endpoints provided by your custom authorization server. Okta recommends using dynamic retrieval and caching of JWKS for performance, with periodic checks for updates and a failsafe for rapid key regeneration.
+
+## Manage the user credential lifecycle
+
+For apps, the choice of token storage strategy and sign-in model significantly impacts security and user experience.
+
+### Token storage
+
+There are several models that you can use to manage tokens in your app:
+
+* Front channel OAuth model
+* Backend For Frontend (BFF)
+* Demonstrating Proof of Possession (DPoP)
+
+The optimal choice depends on the required levels of access control and the desired scalability of your app.
+
+#### Front channel OAuth model
+
+In a front-channel OAuth model, the app handles all authentication directly, and security tokens are stored in cookies, browser storage, or computer memory.
```

Comment:
This is the first mention of "security token" .... unless this is a granular thing outside of access/ID tokens, I suggest we use the access/ID token. If "security token" is a correct term, we should define it clearly.